reference/3.0/Microsoft.PowerShell.Management/Show-ControlPanelItem.md

```diff
@@ -1,74 +1,96 @@
+keywords: powershell,cmdlet
+locale: en-us
+Module Name: Microsoft.PowerShell.Management
+ms.date: 04/22/2019
+online version: http://go.microsoft.com/fwlink/?LinkID=219983
+schema: 2.0.0
+title: Show-ControlPanelItem
 ---
-
 # Show-ControlPanelItem
+
 ## SYNOPSIS
 Opens control panel items.
+
 ## SYNTAX
 
 ### RegularName (Default)
+
 ```
 Show-ControlPanelItem [-Name] <String[]> [<CommonParameters>]
 ```
 
 ### CanonicalName
+
 ```
 Show-ControlPanelItem -CanonicalName <String[]> [<CommonParameters>]
 ```
 
 ### ControlPanelItem
+
 ```
 Show-ControlPanelItem [[-InputObject] <ControlPanelItem[]>] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-The **Show-ControlPanelItem** cmdlet opens control panel items on the local computer.
-You can use it to open control panel items by name, category, or description, even on systems that do not have a user interface, and you can pipe control panel items from Get-ControlPanelItem to **Show-ControlPanelItem**.
 
-**Show-ControlPanelItem** searches only the control panel items that can be opened on the system.
-On computers that do not have Control Panel or File Explorer, **Show-ControlPanelItem** searches only control panel items that can open without these components.
+The `Show-ControlPanelItem` cmdlet opens control panel items on the local computer. You can use it
+to open control panel items by name, category, or description, even on systems that do not have a
+user interface. You can pipe control panel items from the `Get-ControlPanelItem` cmdlet to
+`Show-ControlPanelItem`.
+
+`Show-ControlPanelItem` searches only control panel items that can be opened on the system. On
+computers that do not have **Control Panel** or **File Explorer**, `Show-ControlPanelItem` searches
+only control panel items that can open without these components.
+
+This cmdlet was introduced in Windows PowerShell 3.0 and works on Windows 8, Windows Server 2012,
+and higher versions.
 
-This cmdlet is introduced in Windows PowerShell 3.0.
-It works only on Windows 8 and Windows Server 2012.
-Because this cmdlet requires a user interface, it does not work on Server Core installations of Windows Server.
 ## EXAMPLES
 
-### Example 1: Open a Control Panel Item
-```
-PS C:\> Show-ControlPanelItem -Name AutoPlay
-```
+### Example 1: Show a control panel item
+
+This example launches the **AutoPlay** control panel item.
 
-### Example 2: Pipe a control panel item to Show-ControlPanelItem
+```powershell
+Show-ControlPanelItem -Name "AutoPlay"
 ```
-PS C:\> Get-ControlPanelItem -Name "Windows Firewall" | Show-ControlPanelItem
+
+### Example 2: Pipe a control panel item to this cmdlet
+
+This example opens the **Windows Defender Firewall** control panel item on the local computer.
+The name of the Windows Firewall control panel item has changed over the versions of Windows. This
+example uses a wildcard pattern to find the control panel item.
+
+```powershell
+Get-ControlPanelItem -Name "*Firewall" | `Show-ControlPanelItem`
```

Comment:
Don't think the codefences are intended to remain around **Show-ControlPanelItem** inside this codefence.

Reply:
Fixed